authentication/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+from .utils import FirebaseAPI
+from .models import User
+from rest_framework.exceptions import ParseError
+from drf_yasg.utils import swagger_serializer_method
+from django.conf import settings
+from django.contrib.auth import get_user_model
```

Comment:
Remove all unused imports. And instead of calling `get_user_model` everytime can't we use the User imported from `.models`?

Reply:
@nishantwrp This was recently added to pylint (Refer https://github.com/PyCQA/pylint-django/issues/244 and https://github.com/PyCQA/pylint-django/issues/278). Also, this is mentioned in the [Django docs](https://docs.djangoproject.com/en/3.1/topics/auth/customizing/#referencing-the-user-model) - `Instead of referring to User directly, you should reference the user model using django.contrib.auth.get_user_model(). This method will return the currently active user model – the custom user model if one is specified, or User otherwise`.

That was the reason for the lint errors in https://github.com/IIT-BHU-InstiApp/workshops-app-backend/pull/52.

Reply:
@nb9960 Just remove the unused imports. Everything else looks good!